stdlib/concurrent/futures/thread.pyi

```diff
@@ -1,23 +1,61 @@
 import queue
 import sys
-from typing import Any, Callable, Generic, Iterable, Mapping, Tuple, TypeVar
+import threading
+import weakref
+from collections.abc import Iterable, Mapping, Set
+from concurrent.futures import _base
+from typing import Any, Callable, Tuple
 
-from ._base import Executor, Future
+_threads_queues: Mapping[Any, Any]
+_shutdown: bool
+_global_shutdown_lock: threading.Lock
 
-if sys.version_info >= (3, 7):
-    from ._base import BrokenExecutor
-    class BrokenThreadPool(BrokenExecutor): ...
+def _python_exit() -> None: ...
 
 if sys.version_info >= (3, 9):
     from types import GenericAlias
 
-_S = TypeVar("_S")
+class _WorkItem(object):
```

Comment:
`_WorkItem` was generic before, leading to potentially more exact types. Any reason to change that?

Reply:
I'm still not familiar with the use of Generics for typing, so I mistakenly changed it.
The `proccess.pyi` file also has the `_WorkItem` class among others, should I change their definitions to use a Generic type?

Reply:
If want to try that, it would be appreciated, but as it didn't exist before you don't need to.